import_dataset/schema_less_guide.md

```diff
@@ -0,0 +1,303 @@
+# Data Commons Schema-less Guide
```

Comment:
this file is also missing the required header. here's an example from another file on the site. 
---
layout: default
title: Customize UI
parent: Custom Data Commons
published: true
---

we should also add an index.md file for the site "section". sorry for not catching this earlier

Reply:
added a header! ptal